### PR TITLE
Allow Visual Studio 2019 detection in installer

### PR DESCRIPTION
**Pull request by cmourglia**
Not exactly sure about the validity of this modification (never developed Visual extensions), but this seems to work on my machine™.

### CODE REVIEW

**Comment by Nightro** [comment]
Hey! Thanks for that, I'll check this out in a week or so (work is pretty crazy busy atm) but thank you! I'll do an official release.

File: code/source.extension.vsixmanifest [hunk 1]
```diff
@@ -13,14 +13,14 @@
...
-    <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[14.0,16.0)" />
+    <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[14.0,17.0)" />
   </Installation>
   <Dependencies>
     <Dependency Id="Microsoft.Framework.NDP" DisplayName="Microsoft .NET Framework" d:Source="Manual" Version="[4.5,)" />
     <Dependency Id="Microsoft.VisualStudio.MPF.14.0" DisplayName="Visual Studio MPF 14.0" d:Source="Installed" Version="[14.0]" />
   </Dependencies>
   <Prerequisites>
-    <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[15.0,16.0)" DisplayName="Visual Studio core editor" />
+    <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[15,17.0)" DisplayName="Visual Studio core editor" />
```
Comment (josdemmers) [inline code/source.extension.vsixmanifest:23]:
```suggestion
    <Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[15.0,17.0)" DisplayName="Visual Studio core editor" />
```

**Comment by josdemmers** [comment]
Edited the latest setup to make it work with 2019. There was a small modification needed on the pull request.

In the file **extension.vsixmanifest** I now use the following lines:
`<InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[14.0,17.0)" />`
and
`<Prerequisite Id="Microsoft.VisualStudio.Component.CoreEditor" Version="[15.0,17.0)" DisplayName="Visual Studio core editor" />`

In the file **catalog.json** I now use the following:
`"Microsoft.VisualStudio.Component.CoreEditor":"[15.0,17.0)"`

**Review by Nightro — APPROVED** [review]
Upgrading release to support 2019, thanks for this.

**Pull request merged**